src/common/translate.js

```diff
@@ -60,7 +60,7 @@ const sendRequestToGoogle = async (word, sourceLang, targetLang) => {
   resultData.resultText = result.data.sentences.map(sentence => sentence.trans).join("");
   if (result.data.dict) {
     resultData.candidateText = result.data.dict
-      .map(dict => `${dict.pos}${dict.pos != "" ? ": " : ""}${dict.terms.join(", ")}\n`)
+      .map(dict => `${dict.pos}${dict.pos != "" ? ": " : ""}${dict.terms !== undefined?dict.terms.join(", "):""}\n`)
```

Comment:
Hello guys, I think `dict.terms?.join(", ") || ""` would be simpler.

Reply:
Hi @phuctv95, yes that one is simpler, but the old logic (check undefined) is more reliable.

Reply:
Hi @phuctv95, see the document about [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining).
I believe if we use the optional chaining it may break stability on older versions of browsers. 

but if @sienori confirms, we can change it to that style.

Reply:
about the second question, I agree with you, but based on Google's API response, they should not change the value to something else (we can trust them), so `dict.terms` should be an array of strings.

Reply:
That makes sense to me, thank you for the analysis. So I think keeping using `!== undefined` like current should be totally fine.